### ISSUE TITLE
Suggestion: Add a 5th Tree Farming Plot at Lvl99

### ISSUE THREAD

**Issue by Valle94** [MelvorIdle/melvoridle.github.io#512]
I think it would be nice to have a fifth tree plot at lvl99 to enable growing 1 of each type of tree, possibly for 2m (the same cost of the highest tier gathering upgrades). Both allotments and herbs have plots at least equal to the number of various seeds, and my slight OCD would like trees to be the same. 

Pros:

- Helps solve my OCD by enabling each seed to be farmed simultaneously.

- Wouldn't disrupt farming exp (not including virtual levels) as it wouldn't unlock until 99 anyways.

- Farming is not the primary source of logs, so wouldn't disrupt the gathering balance of WC/Firemaking/Fletching.

Cons:

- It might disrupt the Mastery Pool balance when going to 99 Mastery in all seeds, though I haven't actually done the math. 

- If there is intent to add new seeds in the future (tree or otherwise), adding a spot strictly for this reason could set a bad precedent. 

- Could possibly confuse new/unfamiliar players who would question why something like that unlocks only once you've "completed" the skill. This I think is quite minor though, because you wouldn't be able to see the lvl99 plot until the lvl80 plot has been purchased, at which point one would hope to be a reasonably experienced player. 

Thanks for reading my suggestions. Also I know it's been mentioned in another "Issue" submission, but Oak Tree Seeds are auto-sorting out of order in the bank, and certainly qualifies as mildly infuriating.
